python/understack-workflows/understack_workflows/bmc_sushy.py

```diff
@@ -0,0 +1,15 @@
+from sushy import Sushy
+
+from understack_workflows.bmc_password_standard import standard_password
+from understack_workflows.helpers import credential
+
+
+def bmc_sushy_session(ip_addr, username="root", password=None):
+    """Get a Sushy session for given IP address using standard password."""
+    url = f"https://{ip_addr}"
+
+    if password is None:
+        master_secret = credential("bmc_master", "key")
+        password = standard_password(ip_addr, master_secret)
+
+    return Sushy(url, username=username, password=password, verify=False)
```

Comment:
Why this file vs the one that has a BMC class that does the same thing?

Reply:
Yes this shouldn't be here, removed.